### ISSUE TITLE
🎨  [Application design]  Layout of the Application

### ISSUE THREAD

**Issue by IanOdhiambo9** [italanta/elewa#24]
## Application layout

When a user logs in, they are navigated to the application homepage (Dashboard). Here they can view an analysis of all their bots and all the bots they are building and go to their builder.

---

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user I wish to view a navigation pane to other features of the application and a toolbar with my user menu.

### Description

The application layout at the moment looks like below:

<img width="1432" alt="Screenshot 2022-08-03 at 15 58 30" src="https://user-images.githubusercontent.com/29625264/182614288-6ca1fce3-e9a6-4afa-bb22-c4fa088cb401.png">

From here I can :
1. Interact with the sidenav at the left which contains my navigation.
2. Interact with the user component which contains my user menu.
3. Interact with the application pages which contain the various features of the application.

We have prepared a new design to improve the current UI as shown below:

<img width="1440" alt="Screenshot 2022-08-03 at 16 06 09" src="https://user-images.githubusercontent.com/29625264/182623978-582d82fe-b4da-402c-a0a6-eaa2260cb937.png">

Your assignment, make it look like that picture. Ignore the sections filled in gray colour for now as those are in another feature.

### Technical Analysis

This original page layout is located in libs/elements/layout/page-convl/src/lib/components. The functionality is spread across the different components in this library.

For this feature you'll have to:
```markdown

1. Edit the convl-sidemenu component to match the new design.
2. Edit the convl-navbar component to match the new design.
```

### Acceptance Criteria

1. When rendered the component is pixel perfect compared to the image.
2. The component is well structured following the Technical Analysis

**Comment by meshack34** [comment]
I intend to work on this

**Comment by annk1995** [comment]
I intend to work on this issue.

**Comment by ReaganCn** [comment]
Resolved by Pull request #99
